avogadro/core/mesh.h

```diff
@@ -138,14 +142,6 @@ class AVOGADROCORE_EXPORT Mesh
    */
   const Core::Array<Vector3f>& normals() const;
 
-  /**
```

Comment:
I still don't understand why you're removing `numNormals()` which seems like a useful method.

Reply:
But i was unable to find where we were using numnormals() in our actual code?

Reply:
We're not, currently. But since this is a `core` class, it's also API for future development.